inspirehep/modules/records/cli.py

```diff
@@ -354,6 +356,67 @@ def handle_duplicates(remove_no_control_number, remove_duplicates,
     db.session.commit()
 
 
+@click.command()
+@with_appcontext
+def find_arxiv_duplicates(starting_date=None):
+    """Finds arXiv paper created more than once, from a given date.
+
+    Example:
+        >> from datetime import datetime
+        >> date = datetime(2019, 01, 10)
+        >> dups = find_arxiv_duplicates(date)
+        >> dups
+        {'1812.10562': {'recids': [1711898, 1711919], 'wf_ids': [1387687, 1388915]}}
+
+    Args:
+        starting_date (datetime): date from which starting to search workflows
+
+    Returns:
+        (dict): a dictionary having as key the arXiv id and as value a
+        dictionary having two keys: `recids`, which is a list of control
+        numbers of the paper refering to the arXiv id key, and `wf_ids`,
+        which contains the list od ids of the workflows that created the
+        related records.
+    """
+    if not starting_date:
+        starting_date = datetime.min
+
+    workflows_ids = [
+        wf_id[0] for wf_id in WorkflowObjectModel.query.
+            with_entities(WorkflowObjectModel.id).
+            filter(WorkflowObjectModel.created > starting_date).
+            filter(WorkflowObjectModel.status == ObjectStatus.COMPLETED).
+            all()
+    ]
+    duplicates = dict()
+
+    with click.progressbar(length=len(workflows_ids), label='Processing workflows') as bar:
```

Comment:
that's a strange way to do it. Why not simply
```suggestion
    with click.progressbar(workflow_ids, label='Processing workflows') as bar:
```
followed by
```
    for wf_id in bar:
```
then you don't have to update the bar by hand.